env/includes/shotgun/all.yml

```diff
@@ -43,3 +43,15 @@ shotgun.publishedfile:
       location: "@common.apps.tk-shotgun-launchfolder.location"
 
   location: "@common.engines.tk-shotgun.location"
+
+shotgun.version:
+  apps:
```

Comment:
We should add the `tk-multi-launchapp` and `tk-multi-publish2` app in here as well. Just like the `shotgun.all` setup